vizro-core/docs/pages/explanation/schema.md

```diff
@@ -0,0 +1,156 @@
+        }
+      ]
+    }
+    ```
+
+=== "maps to this Dashboard"
+    [![Dashboard]][dashboard]
+
+!!! note
+    The Vizro schema is not yet complete. This means that it does not yet define everything that we consider a core and supported functionality of Vizro. The most prominent example of this is the [`CapturedCallable`][vizro.models.types.CapturedCallable] - the objects you insert into many models such as `vm.Graph`, `vm.Table` or `vm.Action`, often into the `figure` argument.
+
+    This does not mean that it cannot be configured in JSON - in fact, in the config above we have used the following JSON:
+
+    ```json
+    {
+      "figure": {
+        "_target_": "histogram",
+        "color": "species",
+        "data_frame": "iris",
+        "x": "sepal_width"
+      },
+      "type": "graph"
+    }
+    ```
+
+    It does mean however that e.g. `figure` does not appear in the schema - and as such is not clearly defined as part of the Vizro framework, although it should. We are working on completing the Vizro schema in the future.
+
+## The role of Pydantic
+
+The Vizro framework is powered by [Pydantic](https://docs.pydantic.dev/latest/) - the most widely used data validation library for Python. Pydantic is acts as the glue that connects Vizro's JSON schema to it's emergent manifestation - a React front end served by a Flask backend, facilitated by Dash.
+
+One core advantage of Pydantic is that it emits a [JSON schema](https://blog.postman.com/what-is-json-schema) - so that our models such as `vm.Page` or `vm.Filter` can easily be translated into a well-defined JSON schema. Beyond this capability, it also has support for custom validation beyond the functionality of the JSON schema.
+
+Sticking with our toy example from above - we could for example define that in the array of strings, consecutive elements have to have more letters than their predecessor. This is of course fairly random, but it shows the power of custom validation for anyone using the Vizro framework. Most of the time when the user "misconfigures" - the error message is clear and concise.
+
+## The grammar of dashboards
+
+How does this all come together? One of the long-term goals of Vizro is to define a so-called **grammar of dashboards**. This means that we complete the Vizro schema and define a unified, implementation independent language to configure dashboards.
+
+At the moment, the Vizro framework serves [Dash](https://github.com/plotly/dash) apps - but this is not a necessary condition. In principle, the mapping of JSON to application/dashboard could be realized in any technology.
+
+## The role of `extra`
+
+Some of our models, e.g. the [`Container`][vizro.models.Container], have an argument called `extra`. This argument allows the user to pass extra arguments directly to the underlying component of the model. In the case of the `Container` this would be the [`dbc.Container`](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/layout/).
```

Comment:
```suggestion
Some of our models, e.g. the [`Container`][vizro.models.Container], have an argument called `extra`. This argument enables the user to pass extra arguments directly to the underlying component of the model. In the case of the `Container`, this would be the [`dbc.Container`](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/layout/).
```